mixery/assets/css/app.css

```diff
@@ -3,3 +3,177 @@
 @import "tailwindcss/utilities";
 
 /* This file is for your main application CSS */
+
+/*
+* INFO: this is the style from wild_duck(ahmed_ayob) 
+* */
+
+@import url("https://fonts.googleapis.com/css2?family=Lexend+Deca:wght@100..900&family=Noto+Sans+Arabic:wght@100..900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
```

Comment:
this isnt conform with eu data protection laws :3 (gotta proxy though server)

Reply:
does google does that ?? 
i have heard this before so i should notn't use google fonts ??

Reply:
you can use it , but not directly - you can probably just proxy the request through our own backend , or downlaod it and include the font in your own assets

Reply:
that is a good one